src/pages/UserPage/UserPage.tsx

```diff
@@ -29,6 +35,23 @@ const UserPage: FC = () => {
 
   const userNameString = `${user.user.forename} ${user.user.surname}`
 
+  const handleArchiveModal = () => {
+    const isMainUser = some(user?.roles, (mainUser) => mainUser?.is_root)
+
+    !isMainUser &&
```

Comment:
**[blocker]**

This behaviour does not match the behaviour described in WIKI. Currently we are blocking the archiving of the user, if he has some role with "is_root" true, but according to wiki we should allow these types of users to be archived, unless we would be left with 0 users with that "is_root" role, after archiving. 

-- https://github.com/keeleinstituut/tv-tolkevarav/wiki/5.-Autoriseerimine,--kasutajate-lisamine-s%C3%BCsteemi-(Kasutajate-ja-rollide-haldus)/5d3d608250967b6b0b33de08da0360ed9491d22c#5-archiving-the-account-added-2601

-- Some extra discussion about it in Slack -- https://pundar.slack.com/archives/C053FD55ATT/p1687952259793729

Basically the idea here is that we don't do any check on FE and just show an error Alert, if BE returns the error, that this account can't be archived

Reply:
Yes, currently it is blocked by BE task #394, waiting for deploy